py_ecc/fields/field_elements.py

```diff
@@ -201,7 +201,7 @@ class FQP(object):
 
     def __init__(self,
                  coeffs: Sequence[IntOrFQ],
-                 modulus_coeffs: Sequence[IntOrFQ] = None) -> None:
+                 modulus_coeffs: Sequence[IntOrFQ] = []) -> None:
```

Comment:
This is to handle a separate error where `len(modulus_coeffs)` is called in line 208 below. This is not possible with `None` as it has no `__len__`. The reason for having a default argument here is for simple `type(self)` calls.